common-lib/src/main/scala/com/gu/mediaservice/lib/cleanup/SupplierProcessors.scala

```diff
@@ -170,10 +170,14 @@ trait GettyProcessor {
 }
 
 object GettyXmpParser extends ImageProcessor with GettyProcessor {
-  val excludeFrom = List("newspix international")
+  // including 'newspix internation' because they're sending us lots with that in the credit
+  val excludeFromPatterns = List("newspix international", "newspix internation", "i-images")
+    .map(ex => s"(?i)$ex".r)
```

Comment:
I think the (?i) is scala's version of "/i", i.e. case insensitive 

http://daily-scala.blogspot.co.uk/2010/01/regular-expression-2-rest-regex-class.html

So I think this just creates a list of 3 case insensitive regexs